src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -336,8 +336,8 @@ private void testAddAction() throws Exception {
 
         newSession.setFeedbackSessionName("responses cant be seen my students 1 #");
         // start time in past
-        newSession.setStartTime(TimeHelper.convertToDate("2012-05-01 6:00 AM UTC"));
-        newSession.setEndTime(TimeHelper.convertToDate("2017-32-12 1:59 AM UTC"));
+        newSession.setStartTime(TimeHelper.convertToDate("2012-05-01 6:00 AM +0000"));
+        newSession.setEndTime(TimeHelper.convertToDate("2030-12-12 1:59 AM +0000"));
```

Comment:
Maybe use 2037 to standardise with this? https://github.com/TEAMMATES/teammates/pull/7028#issue-113869667

Reply:
Okay